android/app/src/main/java/org/fossasia/openevent/adapters/DayScheduleAdapter.java

```diff
@@ -29,18 +30,28 @@
  */
 public class DayScheduleAdapter extends BaseRVAdapter<Session, DayScheduleViewHolder> implements StickyRecyclerHeadersAdapter<HeaderViewHolder> {
 
+    public static final int CODE_UNDO_ADDED = 1;
+    public static final int CODE_BLANK = 0;
+    public static final int CODE_UNDO_REMOVED = 2;
+
     private Context context;
     private String eventDate;
+    private DayScheduleCallback dayScheduleCallback;
 
     private RealmDataRepository realmRepo = RealmDataRepository.getDefaultInstance();
 
     private ArrayList<String> tracks = new ArrayList<>();
     private List<Session> copyOfSessions = new ArrayList<>();
 
-    public DayScheduleAdapter(List<Session> sessions, Context context) {
+    public interface DayScheduleCallback {
```

Comment:
Please rename to a proper name, like OnBookmarkSelectedListener

Reply:
Also this is used in places other than DayScheduleAdapter, so move to its own class